src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -577,14 +578,18 @@ private Cluster parseCluster(Row row) throws IOException {
           ? objectMapper.readValue(row.getString("properties"), ClusterProperties.class)
           : ClusterProperties.builder().withJmxPort(Cluster.DEFAULT_JMX_PORT).build();
 
+    LocalDate lastContact = row.getTimestamp("last_contact") == null
+        ? new java.sql.Date(new Date().getTime()).toLocalDate()
```

Comment:
when null it should be `lastContact = LocalDate.MIN`

(subsequent jmx calls will update it correctly accordingly…)